drivers/cpl/nuopc/glc_import_export.F90

```diff
@@ -76,12 +87,13 @@ module glc_import_export
 contains
 !===============================================================================
 
-  subroutine advertise_fields(gcomp, num_icesheets_in, rc)
+  subroutine advertise_fields(gcomp, cism_evolve, num_icesheets_in, rc)
```

Comment:
I sent an email to @mvertens about this, but I would like to keep the functionality of the coupler downscaling data in NO-EVOLVE going forward. If this is incompatible with the runoff changes, please let me know.

Reply:
Ah, I think I figured out that it probably has to do with temperature and salinity on ocean depth levels. Those are definitely not needed in No_evolve mode and should be inside of a check. But the 2D surface temp and mass balance should be advertised in both cases.

Reply:
@Katetc - thanks for catching this.  I have pushed the fix back.